main.go

```diff
@@ -1,9 +1,14 @@
 package main
 
-import "github.com/terrariumcloud/terrarium/cmd"
+import (
+	"log"
+
+	"github.com/terrariumcloud/terrarium/cmd"
+)
 
 var buildInformationVersion string
 
 func main() {
 	cmd.Execute(buildInformationVersion)
+	log.Println(buildInformationVersion)
```

Comment:
make that line first otherwise you will never see it as execute is blocking call